### PR TITLE
Fix Rollout issue

### PR DESCRIPTION
**Pull request by venkayya**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -237,6 +238,19 @@ overview of a manual upgrade of a Kubernetes Engine cluster.
...
+
+If Running into an error while executing `make` commands similar to the below example output
+```console
+Waiting for rollout to finish: 1 of 3 updated replicas are available...
+error: deployment "es-master" exceeded its progress deadline
+```
+The solution would be to double check if the region is set correctly and not zone.
+Run this command to set the region.
+```console
+gcloud config set compute/region us-east1
```
Comment (chrislovecnm) [inline README.md]: we do not want to list a specific region.  Let the user decide.

Comment (venkayya) [reply README.md]: replaced default region with generic name

File: README.md [hunk 2]
```diff
@@ -237,6 +238,19 @@ overview of a manual upgrade of a Kubernetes Engine cluster.
...
+
+If Running into an error while executing `make` commands similar to the below example output
+```console
+Waiting for rollout to finish: 1 of 3 updated replicas are available...
+error: deployment "es-master" exceeded its progress deadline
+```
+The solution would be to double check if the region is set correctly and not zone.
+Run this command to set the region by replacing REGION_NAME with desired region.
+```console
+gcloud config set compute/region <REGION_NAME>
```
Comment (chrislovecnm) [inline README.md]: Do we check that this is set in the create script?  We should.  Missed this in the initial review.

Comment (chrislovecnm) [inline README.md]: https://github.com/GoogleCloudPlatform/gke-rolling-updates-demo/blob/master/blue-green-upgrade/cluster_ops.sh#L69

Should catch this problem. How did it not?

**Comment by chrislovecnm** [comment]
Test this please

**Comment by venkayya** [comment]
test this please

**Pull request closed without merging**